frontend/src/old-pages/Configure/Storage.tsx

```diff
@@ -418,9 +422,7 @@ function EbsSettings({
             <FormField errorText = {volumeErrors}>
               <Input
                 disabled={editing}
-                style={{marginTop: 10}}
-                // @ts-expect-error TS(2322) FIXME: Type '"decimal"' is not assignable to type 'Type |... Remove this comment to see the full error message
-                type="decimal"
+                type={'decimal' as InputProps.Type}
```

Comment:
issue(blocking): I don't see `decimal` among the supported types. They do list `decimal` as `inputMode`, did you mean to specify the input mode?

[reference](https://polaris.a2z.com/components/input/?tabId=api)

Reply:
Good catch -- I believe I was more looking for `type='number'` but I updated that and added the input mode as well. :+1: